internal/provisioner/model/model.go

```diff
@@ -254,6 +255,18 @@ type ContourSpec struct {
 	// to only watch these set of namespaces
 	// default is nil, contour will watch resource of all namespaces
 	WatchNamespaces []string
```

Comment:
Will change this to original v1.Namespace type in a follow up PR.
I think it make more sense to keep model and types consistent, and put conversion function inside model

Reply:
Cool let me change it here